src/elementary/src/toEndfReactionNumber.hpp

```diff
@@ -0,0 +1,45 @@
+#ifndef NJOY_ELEMENTARY_TOENDFREACTIONNUMBER
+#define NJOY_ELEMENTARY_TOENDFREACTIONNUMBER
+
+// system includes
+
+// other includes
+#include "elementary/src/split.hpp"
+#include "elementary/ParticlePairID.hpp"
+#include "elementary/ParticleTupleID.hpp"
+#include "elementary/ReactionID.hpp"
+
+namespace njoy {
+namespace elementary {
+
+  /**
+   *  @brief Return the MT number for the reaction, if one can be derived
+   *
+   *  @param[in] identifier   the reaction identifier
+   */
+  int toEndfReactionNumber( const ReactionID& identifier ) {
+
+    // treat special cases
+
+    // other cases
+

```

Comment:
What about these? Are you intending to fill these in?

Reply:
It's not done yet. There was enough to merit a pull request though. I'm currently working on a ReactionType that will help link ReactionID to an MT number, if one can be derived.